integreat_cms/core/circleci_settings.py

```diff
@@ -28,6 +28,8 @@
 LINKCHECK_DISABLE_LISTENERS = True
 #: Enable logging of all entries from the messages framework
 MESSAGE_LOGGING_ENABLED = True
+# Disable background tasks during testing
+INTEGREAT_CMS_BACKGROUND_TASKS_ENABLED = True
```

Comment:
```suggestion
INTEGREAT_CMS_BACKGROUND_TASKS_ENABLED = False
```

Reply:
Not only the variable name, but also the value needs to be changed :see_no_evil:

Reply:
😅 wow. sometimes you actually need to mind every detail in a review. sorry for the delay and thanks for the awesome review 🎉